docker-compose.yml

```diff
@@ -45,25 +45,24 @@ services:
             # EMAIL_HOST_PASSWORD: smtp_account_password
             # EMAIL_PORT: 587 # optional
             # EMAIL_USE_TLS: True # optional
-            
-            # ENABLE_ADMIN_ACCESS_USER_PASS: True # set to True to enable access to the /admin/ Django backend via your username and password
+
+            # set to True to enable access to the /admin/ Django backend via your username and password
+            ENABLE_ADMIN_ACCESS_USER_PASS: 'true'
```

Comment:
This is disabled by default for additional security. What's your reason for enabling it?